documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM1.adoc

```diff
@@ -0,0 +1,26 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM1
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM1
+
+== New shortcuts in Fuse perspective
+
+Shortcuts for Java, Launch and Debug configuration and basic navigation are now enabled by default with Fuse perspective.
+
+The result is a set of actions now available in the Toolbar:
```

Comment:
"As a result a set of actions are now available in the toolbar:" ?

Reply:
"actions" are too much Eclipse-specific vocabulary?
I will use buttons then

Reply:
um....thats ok. But that wasn't the intend of my suggestion ;)